Shared/Models/Eatery.swift

```diff
@@ -259,16 +259,41 @@ extension Eatery {
         let events = eventsByName(onDayOf: Date())
         for event in events {
             for (_, items) in event.value.menu.data {
-                if items.contains(where: { item in
-                    DefaultsKeys.isFavoriteFood(item.name)
-                }) {
-                    return true
-                }
+                if items.contains(where: { DefaultsKeys.isFavoriteFood($0.name) }) { return true }
+            }
+        }
+        if let eatery = self as? CampusEatery, let expandedMenu = eatery.expandedMenu {
+            for (_, items) in expandedMenu.data {
+                if items.contains(where: { DefaultsKeys.isFavoriteFood($0.name) }) { return true }
             }
         }
         return false
     }
 
+    func getFavorites() -> [String] {
+        var favorites = [String]()
+        let events = self.eventsByName(onDayOf: Date())
```

Comment:
```suggestion
        let events = eventsByName(onDayOf: Date())
```

You don't need `self`.